dfx.json

```diff
@@ -33,8 +33,18 @@
       "dependencies": [
         "ledger"
       ]
+    },
+    "assets": {
+      "type": "assets",
+      "frontend": {
+        "entrypoint": "assets/index.html"
+      },
+      "source": [
+        "assets/"
+      ]
     }
   },
+  "dfx": "0.14.1",
```

Comment:
lets remove this, we enforce the compiler using `DFX_MOC_PATH`. or do you need it for the test scripts or smth?

Reply:
I think only assets canister code related to this setting, but if you use latest dfx it's ok, we can remove this